client/web/src/marketing/SurveyUseCaseForm.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react'
+
+import classNames from 'classnames'
+
+import { Button, FlexTextArea, TextArea } from '@sourcegraph/wildcard'
+
+import { SurveyUseCaseCheckbox } from './SurveyUseCaseCheckbox'
+import { Toast } from './Toast'
+
+import styles from './SurveyUseCaseForm.module.scss'
+
+export const OPTIONS = [
+    {
+        id: 'understandCodebase',
+        labelValue: 'Understand a new codebase',
+    },
+    {
+        id: 'fixSecurityVulnerability',
+        labelValue: 'Fix security vulnerabilities',
+    },
+    {
+        id: 'reuseCode',
+        labelValue: 'Reuse code',
+    },
+    {
+        id: 'respondToIncidents',
+        labelValue: 'Respond to incidents',
+    },
+    {
+        id: 'ImproveCodeQuality',
+        labelValue: 'Improve code quality',
+    },
+    {
+        id: 'other',
+        labelValue: 'Other',
+    },
+]
+
+interface FormStateType {
+    moreSharedInfo: string
+    otherUseCase: string
+    useCases: string[]
+}
+
+interface SurveyUseCaseFormProps {
+    onDismiss: () => void
+    handleDone: (props: FormStateType) => void
+}
+
+export const SurveyUseCaseForm: React.FunctionComponent<SurveyUseCaseFormProps> = ({ onDismiss, handleDone }) => {
+    const [useCases, setUseCases] = useState<string[]>([])
+    const [otherUseCase, setOtherUseCase] = useState<string>('')
+    const [moreSharedInfo, setMoreSharedInfo] = useState<string>('')
+
+    const handleSubmit = (): void => {
+        handleDone({
+            useCases,
+            otherUseCase,
+            moreSharedInfo,
+        })
+    }
+
+    const handleSelectUseCase = (value: string): void => {
+        if (useCases.includes(value)) {
+            setUseCases(current => current.filter(instance => instance !== value))
+            return
+        }
+        setUseCases(current => [...current, value])
+    }
+
+    return (
+        <Toast
+            toastBodyClassName={styles.toastBody}
+            subtitle={<span className={styles.toastSubtitle}>You are using sourcegraph to...</span>}
+            cta={
+                <div className="mb-2">
+                    <div className={styles.checkWrap}>
+                        {OPTIONS.map(({ id, labelValue }) => (
+                            <SurveyUseCaseCheckbox
+                                onChange={() => handleSelectUseCase(id)}
+                                key={id}
+                                label={<span className={classNames('ml-1', styles.checkboxLabel)}>{labelValue}</span>}
+                            />
+                        ))}
```

Comment:
```suggestion
            subtitle={<span id="usecase-group" className={styles.toastSubtitle}>You are using sourcegraph to...</span>}
            cta={
                <div className="mb-2">
                    <div className={styles.checkWrap}>
                        {OPTIONS.map(({ id, labelValue }) => (
                            <SurveyUseCaseCheckbox
                                onChange={() => handleSelectUseCase(id)}
                                key={id}
                                label={<span id={id} className={classNames('ml-1', styles.checkboxLabel)}>{labelValue}</span>}
                                aria-labelledby={`usecase-group ${id}`}
                            />
                        ))}
```

I think we'll have to do something like this to make it accessible. Note the ids on each element + subtitle and aria-labelledby